tests/e2e/scenarios/hello-world/test.yml

```diff
@@ -0,0 +1,74 @@
+- name: Hello World Test
+  hosts: all
+  roles:
+    - e2e.tests.skupper_test_images
+  tasks:
+    - name: Main task block
+      block:
+        - name: Environment shakeout
+          ansible.builtin.include_role:
+            name: e2e.tests.env_shakeout
+
+        - name: Generating the namespaces
+          ansible.builtin.include_role:
+            name: e2e.tests.generate_namespaces
+
+        - name: Creating Skupper resources on west namespace
+          skupper.v2.resource:
+            path: "{{ item }}"
+            namespace: "{{ namespace_prefix }}-{{ namespace_name }}"
+            kubeconfig: "{{ kubeconfig }}"
+          with_items:
+            - - "{{ playbook_dir }}/resources/west/frontend.yml"
+            - - "{{ playbook_dir }}/resources/west/site.yml"
+            - - "{{ playbook_dir }}/resources/west/listener.yml"
+          when:
+            - "'west' in inventory_hostname"
+
+        - name: Creating Skupper resources on east namespace
+          skupper.v2.resource:
+            path: "{{ item }}"
+            namespace: "{{ namespace_prefix }}-{{ namespace_name }}"
+            kubeconfig: "{{ kubeconfig }}"
+          with_items:
+            - - "{{ playbook_dir }}/resources/east/backend.yml"
+            - - "{{ playbook_dir }}/resources/east/site.yml"
+            - - "{{ playbook_dir }}/resources/east/connector.yml"
+          when:
+            - "'east' in inventory_hostname"
+
+        - name: Issue a Skupper access token from west namespace
+          skupper.v2.token:
+            namespace: "{{ namespace_prefix }}-{{ namespace_name }}"
+            name: west-grant
+            redemptions_allowed: 1
+            kubeconfig: "{{ kubeconfig }}"
+          register: west
+          when:
+            - "'west' in inventory_hostname"
+
+        - name: Apply token to east site
+          skupper.v2.resource:
+            def: "{{ hostvars['west']['west']['token'] }}"
+            namespace: "{{ namespace_prefix }}-{{ namespace_name }}"
+            kubeconfig: "{{ kubeconfig }}"
+          when:
+            - "'east' in inventory_hostname"
+
+        - name: Testing the connectivity between the sites
+          ansible.builtin.include_role:
+            name: e2e.tests.run_curl
+          when:
+            - "'west' in inventory_hostname"
+
+      always:
+        - name: Deleting the namespaces
+          kubernetes.core.k8s:
+            state: absent
+            api_version: v1
+            kind: Namespace
+            label_selectors:
+              - "e2e.id={{ generate_namespaces_namespace_label }}"
```

Comment:
Maybe better to use a more specific value for `generate_namespaces_namespace_label` as it is currently `e2e`.

Reply:
Done under: 425bde40